frontend/src/views/Transactions/_schema.js

```diff
@@ -24,28 +24,48 @@ export const transactionsColDefs = (t) => [
       const transactionType = params.data.transactionType
       const prefix = prefixMap[transactionType] || ''
       return `${prefix}${params.data.transactionId}`
+    },
+    filterParams: {
+      buttons:["clear"],
     }
   },
   {
     colId: 'transactionType',
     field: 'transactionType',
     headerName: t('txn:txnColLabels.type'),
     valueFormatter: spacesFormatter,
+    filter: true, // Enable filtering
+    filterParams: {
+      textFormatter: (value) => value.replace(/\s+/g, '').toLowerCase(),
+      textCustomComparator: (filter, value, filterText) => {
+        // Remove spaces and convert both to lowercase for comparison
+        const cleanFilterText = filterText.replace(/\s+/g, '').toLowerCase();
+        const cleanValue = value.replace(/\s+/g, '').toLowerCase();
+        return cleanValue.includes(cleanFilterText);
+      },
+      buttons:["clear"],
+    },
     width: 222
   },
   {
     colId: 'fromOrganization',
     field: 'fromOrganization',
     headerName: t('txn:txnColLabels.organizationFrom'),
     minWidth: 300,
-    flex: 2
+    width: 300,
```

Comment:
Are you sure we want to use a fixed width? Can you make sure the table still grows / shrinks appropriately?

Reply:
Having flex property is not providing this elasticity. We might need to add the corresponding styling based on the view port and have the application more compatible with different devices.

Reply:
I'm asking as I recently added those flex attributes to allow the table to grow correctly on larger screens and was wondering if this broke that.